src/ontology/cob.Makefile

```diff
@@ -103,6 +103,25 @@ $(COB_ANNOTATIONS): $(TMPDIR)/cob-annotations.ttl
 	--annotation owl:versionInfo $(TODAY) \
 	--output $@
 
+########################################
+# -- USAGES --
+########################################
+
+# USAGE_SOURCES = ontobee ubergraph bioportal
+USAGE_SOURCES = ontobee ubergraph
+
+all_usages: $(patsubst %, reports/summary-cob-usages-%.tsv, $(USAGE_SOURCES))
```

Comment:
Would you like this to run in the context of an existing workflow, eg release?

Reply:
Seconding Nico's question (sorry if I've missed an answer to this elsewhere).